src/Nox.Types/Types/StreetAddress/CountryPostalCodeValidator.cs

```diff
@@ -53,198 +54,159 @@ public static bool IsValid(string countryCode, string? postalCode)
-        ("MH",@"^\d{5}$"),
-        ("MQ",@"^972\d{2}$"),
-        ("YT",@"^976\d{2}$"),
-        ("FM",@"^\d{5}(-{1}\d{4})$"),
-        ("MX",@"^\d{5}$"),
-        ("FM",@"^\d{5}$"),
-        ("MD",@"^[Mm][Dd][- ]{0,1}\d{4}$"),
-        ("MC",@"^980\d{2}$"),
-        ("MN",@"^\d{5}$"),
-        ("ME",@"^\d{5}$"),
-        ("MS",@"^[Mm][Ss][Rr]\s{0,1}\d{4}$"),
-        ("MA",@"^\d{5}$"),
-        ("MZ",@"^\d{4}$"),
-        ("MM",@"^\d{5}$"),
-        ("NA",@"^\d{5}$"),
-        ("NP",@"^\d{5}$"),
-        ("NL",@"^\d{4}\s{0,1}[A-Za-z]{2}$"),
-        ("NC",@"^988\d{2}$"),
-        ("NZ",@"^\d{4}$"),
-        ("NI",@"^\d{5}$"),
-        ("NE",@"^\d{4}$"),
-        ("NG",@"^\d{6}$"),
-        ("NF",@"^\d{4}$"),
-        ("MP",@"^\d{5}$"),
-        ("NO",@"^\d{4}$"),
-        ("OM",@"^\d{3}$"),
-        ("PK",@"^\d{5}$"),
-        ("PW",@"^\d{5}$"),
-        ("PA",@"^\d{6}$"),
-        ("PG",@"^\d{3}$"),
-        ("PY",@"^\d{4}$"),
-        ("PE",@"^\d{5}$"),
-        ("PH",@"^\d{4}$"),
-        ("PN",@"^[Pp][Cc][Rr][Nn]\s{0,1}[1][Zz]{2}$"),
-        ("PL",@"^\d{2}[- ]{0,1}\d{3}$"),
-        ("PT",@"^\d{4}$"),
-        ("PT",@"^\d{4}[- ]{0,1}\d{3}$"),
-        ("PR",@"^\d{5}$"),
-        ("RE",@"^974\d{2}$"),
-        ("RO",@"^\d{6}$"),
-        ("RU",@"^\d{6}$"),
-        ("BL",@"^97133$"),
-        ("SH",@"^[Ss][Tt][Hh][Ll]\s{0,1}[1][Zz]{2}$"),
-        ("MF",@"^97150$"),
-        ("PM",@"^97500$"),
-        ("VC",@"^[Vv][Cc]\d{4}$"),
-        ("SM",@"^4789\d$"),
-        ("SA",@"^\d{5}(-{1}\d{4})?$"),
-        ("SN",@"^\d{5}$"),
-        ("RS",@"^\d{5}$"),
-        ("RS",@"^\d{5}$"),
-        ("SG",@"^\d{2}$"),
-        ("SG",@"^\d{4}$"),
-        ("SG",@"^\d{6}$"),
-        ("SK",@"^\d{5}\s\(\d{3}\s\d{2}\)$"),
-        ("SI",@"^([Ss][Ii][- ]{0,1}){0,1}\d{4}$"),
-        ("ZA",@"^\d{4}$"),
-        ("GS",@"^[Ss][Ii][Qq]{2}\s{0,1}[1][Zz]{2}$"),
-        ("KR",@"^\d{6}\s\(\d{3}-\d{3}\)$"),
-        ("ES",@"^\d{5}$"),
-        ("LK",@"^\d{5}$"),
-        ("SD",@"^\d{5}$"),
-        ("SZ",@"^[A-Za-z]\d{3}$"),
-        ("SE",@"^\d{3}\s*\d{2}$"),
-        ("CH",@"^\d{4}$"),
-        ("SJ",@"^\d{4}$"),
-        ("TW",@"^\d{5}$"),
-        ("TJ",@"^\d{6}$"),
-        ("TH",@"^\d{5}$"),
-        ("TT",@"^\d{6}$"),
-        ("SH",@"^[Tt][Dd][Cc][Uu]\s{0,1}[1][Zz]{2}$"),
-        ("TN",@"^\d{4}$"),
-        ("TR",@"^\d{5}$"),
-        ("TM",@"^\d{6}$"),
-        ("TC",@"^[Tt][Kk][Cc][Aa]\s{0,1}[1][Zz]{2}$"),
-        ("UA",@"^\d{5}$"),
-        ("GB",@"^[A-Z]{1,2}[0-9R][0-9A-Z]?\s*[0-9][A-Z-[CIKMOV]]{2}"),
-        ("US",@"^\b\d{5}\b(?:[- ]{1}\d{4})?$"),
-        ("UY",@"^\d{5}$"),
-        ("VI",@"^\d{5}$"),
-        ("UZ",@"^\d{3} \d{3}$"),
-        ("VA",@"^120$"),
-        ("VE",@"^\d{4}(\s[a-zA-Z]{1})?$"),
-        ("VN",@"^\d{6}$"),
-        ("WF",@"^986\d{2}$"),
-        ("ZM",@"^\d{5}$")
-    });
+    private static readonly ImmutableList<(string CountryCode, string Pattern)> PostalCodesMapping =
+        ImmutableList.CreateRange(new[]
+        {
```

Comment:
@msadikkoseaupa  Besides consolidation, we should keep the ones that do not exist in Nox.Reference and add them to Nox.Reference as well, correct?

Reply:
Actually I did not merge them, instead I completely replaced them from Reference. 
Let me update it. 
So what if same country code has multiple regex from multiple resources